docs/developer_guide/ZeroMQ.md

```diff
@@ -50,7 +50,7 @@ sequenceDiagram
 
 ## Schema of the message
 
-`traffic_simulator::API` sends the request to the simulator. The requests are serialized by using protobuf and use various ports in order to communicate with the simulator.
+The `traffic_simulator::API` sends a request to the simulator. The request is serialized using protobuf and uses the port specified by the ROS Parameter `port` (default is 5555) to communicate with the simulator.
```

Comment:
In effect, the default port is not 5555, because it is overridden in launch file.
You should fix launch file.
https://github.com/tier4/scenario_simulator_v2/blob/master/test_runner/scenario_test_runner/launch/scenario_test_runner.launch.py#L74

Reply:
I changed default ports to `6666`.
Default value is same in launch file and C++ code.